cmd/neofs-node/config.go

```diff
@@ -17,6 +17,7 @@ import (
 
 	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
 	neogoutil "github.com/nspcc-dev/neo-go/pkg/util"
+	utilneogo "github.com/nspcc-dev/neo-go/pkg/util"
```

Comment:
we already have this import aliased with `neogoutil`

Reply:
fixed

Reply:
btw, added a linter for it, we like double importing a little